### ISSUE TITLE
A11Y: Widget with role of menu must have no more than one tabbable element.

### ISSUE THREAD

**Issue by rob-richards** [IBM/carbon-components-react#1714]
> Describe in detail the issue you're having.
I am getting a DAP error because there are more than one tabbable elements in the overflow menu. The tabIndex for all of the `<button>` elements have a `tabIndex` of 0. 

> Is this issue related to a specific component?
Yes - OverflowMenu

> What browser are you working in?
Chrome 70.0.3538.77

> What version of the Carbon Design System are you using?
Latest

Link to component: http://react.carbondesignsystem.com/iframe.html?selectedKind=OverflowMenu&selectedStory=basic&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Link to aria best practices for tabbed navigation: https://www.w3.org/TR/wai-aria-practices-1.1/#kbd_roving_tabindex

<img width="925" alt="screen shot 2019-01-08 at 1 51 03 pm" src="https://user-images.githubusercontent.com/7707326/50852078-9a60fb00-134c-11e9-943c-bf206a836385.png">

As you can see, all of the `button` elements have a `tabIndex` of 0:
<img width="751" alt="screen shot 2019-01-08 at 1 38 20 pm" src="https://user-images.githubusercontent.com/7707326/50852086-9e8d1880-134c-11e9-81c3-2340ba79b071.png">

**Comment by joshblack** [comment]
cc @dakahn

**Comment by dakahn** [comment]
Totally! We'll need to rethink the keyboard navigation implementation for this component. I'll assign myself.

Here's the example navigation
https://www.w3.org/TR/wai-aria-practices-1.1/examples/menubar/menubar-1/menubar-1.html

**Comment by rob-richards** [comment]
@dakahn My team has come across a few components with this same error. You may want to consider creating a higher order component that will set the `focus()` and `tabIndex` of the tabbable elements in a menu.

**Comment by dakahn** [comment]
Looping back on this. DAP no longer throws this error now that the default tabindex for the menuitems is -1. 👍 Happy to reopen if I need to.
